src/main/java/Jelly/commands/DeleteCommand.java

```diff
@@ -0,0 +1,26 @@
+package Jelly.commands;
+
+import Jelly.main.Storage;
+import Jelly.main.TaskList;
+import Jelly.main.Ui;
+
+public class DeleteCommand extends Command {
+
+    private int index;
+
+    public DeleteCommand(int index) {
+        this.index = index;
+    }
+
+    @Override
+    public void execute(TaskList taskList, Ui ui, Storage storage) {
+        if (index <= 0 || index > 100 || index > taskList.size()) {
+            System.out.println("Invalid input");
+        } else if (taskList.get(index - 1) != null) {
+            ui.deleteMessage(taskList.get(index - 1), taskList.size() - 1);
+            taskList.delete(index - 1);
+        } else {
+            System.out.println("Invalid input");
+        }
+    }
```

Comment:
Not really a coding standard issue but perhaps you can try guard clauses to improve readability?

```java
@Override
    public void execute(TaskList taskList, Ui ui, Storage storage) {
        if (index <= 0 || index > 100 || index > taskList.size()) {
            System.out.println("Invalid input");
            return;
        }
        
        if (taskList.get(index - 1) != null) {
            ui.deleteMessage(taskList.get(index - 1), taskList.size() - 1);
            taskList.delete(index - 1);
            return;
        }
        
        System.out.println("Invalid input");
    }
```